module.rb

```diff
@@ -0,0 +1,17 @@
+module MyEnumerable
+  def all?
+    my_each { |item| return true if yield item }
+    true
+  end
+
+  def any?
+    my_each { |item| return false unless yield item }
+    false
+  end
+
```

Comment:
- [ ] You implemented the changes wrongly. The tests below should return, `false`, `false`, `false`, `true` respectively but that's not the case from the screenshot below

  ```
  list = MyList.new(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
  p list
  
  p list.all? { |e| e < 5 }
  p list.all? { |e| e > 5 }
  
  p list.any?(&:zero?)
  p list.any? { |e| e == 5 }
  ```

  ![list](https://user-images.githubusercontent.com/54848363/185609289-60141bd8-2ec6-4fb8-bad2-703ae5179b6d.jpg)

  Kindly pay close attention to the combination of if/unless and the return statements in the enumerable snippet I suggested. The `all?` method should be this:
  ```
  def all?
    my_each { |item| return false unless yield item }
    true
  end
  ```
  The `any?` method should be this:
  ```
  def any?
    my_each { |item| return true if yield item }
    false
  end
  ```